molgenis-questionnaires/src/main/frontend/src/store/actions.js

```diff
@@ -83,21 +85,29 @@ const actions = {
     })
   },
 
-  'AUTO_SAVE_QUESTIONNAIRE' ({commit, state}: VuexContext, formData: Object) {
-    commit('INCREMENT_SAVING_QUEUE')
-    const updatedAttribute = Object.keys(formData).find(key => formData[key] !== state.formData[key]) || ''
-
-    const options = {
-      body: JSON.stringify(formData[updatedAttribute]),
-      method: 'PUT'
-    }
+  'AUTO_SAVE_QUESTIONNAIRE' ({commit, state}: VuexContext, payload: Object) {
+    const {formData, formState} = payload
 
-    return api.post(`/api/v1/${state.questionnaire.meta.name}/${state.questionnaireRowId}/${updatedAttribute}`, options).then(() => {
-      commit('SET_FORM_DATA', formData)
-    }, error => {
-      handleError(commit, error)
-    }).then(() => {
-      commit('DECREMENT_SAVING_QUEUE')
+    const updatedAttribute = Object.keys(formData).find(key => formData[key] !== state.formData[key]) || ''
+    commit('SET_FORM_DATA', formData)
+
+    Vue.nextTick(() => {
+      // TODO To refactor this code without the nextTick method we need to expose the validation
```

Comment:
Consider using one TODO instead of two so that it shows up correctly in e.g. IDE
TODO= ``` Task that will never be solved – “Don`t have time now, will do it later” where later equals never.```. How about creating a GitHub enhancement issue and refer to the URL in your code?

Reply:
already removed